### PR TITLE
More convergence settings in various gameini files

### PR DESCRIPTION
**Pull request by SMarioMan**
I've tested most of my games to find appropriate convergence values. These were my results.

### CODE REVIEW

**Comment by lioncash** [comment]
Can you please rebase these all into one commit?

File: Data/Sys/GameSettings/GSR.ini# RUUE01, RUUJ01, RUUK01, RUUP01 - Animal Crossing Wii [Core] # Values set here will override the main Dolphin settings. [EmuState] # The Emulation State. 1 is worst, 5 is best, 0 is not set. EmulationStateId = 4 EmulationIssues =  [OnLoad] # Add memory patches to be loaded once on boot here. [OnFrame] [ActionReplay] [Video_Hacks] EFBToTextureEnable = False [Video_Stereoscopy] StereoConvergenceMinimum = 381 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# GSRE7S - Smuggler's Run: Warzones
+[Core]
+# Values set here will override the main Dolphin settings.
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId =
```
Comment (phire) [inline Data/Sys/GameSettings/GSR.ini# RUUE01, RUUJ01, RUUK01, RUUP01 - Animal Crossing Wii [Core] # Values set here will override the main Dolphin settings. [EmuState] # The Emulation State. 1 is worst, 5 is best, 0 is not set. EmulationStateId = 4 EmulationIssues =  [OnLoad] # Add memory patches to be loaded once on boot here. [OnFrame] [ActionReplay] [Video_Hacks] EFBToTextureEnable = False [Video_Stereoscopy] StereoConvergenceMinimum = 381]: [According to our wiki](https://wiki.dolphin-emu.org/index.php?title=Smuggler%27s_Run:_Warzones) this game should have an Emulation state of 4.

File: Data/Sys/GameSettings/GSR.ini# RUUE01, RUUJ01, RUUK01, RUUP01 - Animal Crossing Wii [Core] # Values set here will override the main Dolphin settings. [EmuState] # The Emulation State. 1 is worst, 5 is best, 0 is not set. EmulationStateId = 4 EmulationIssues =  [OnLoad] # Add memory patches to be loaded once on boot here. [OnFrame] [ActionReplay] [Video_Hacks] EFBToTextureEnable = False [Video_Stereoscopy] StereoConvergenceMinimum = 381 [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# GSRE7S - Smuggler's Run: Warzones
```
Comment (phire) [inline Data/Sys/GameSettings/GSR.ini# RUUE01, RUUJ01, RUUK01, RUUP01 - Animal Crossing Wii [Core] # Values set here will override the main Dolphin settings. [EmuState] # The Emulation State. 1 is worst, 5 is best, 0 is not set. EmulationStateId = 4 EmulationIssues =  [OnLoad] # Add memory patches to be loaded once on boot here. [OnFrame] [ActionReplay] [Video_Hacks] EFBToTextureEnable = False [Video_Stereoscopy] StereoConvergenceMinimum = 381]: Also, this file is completely screwed, contains another file in the filename.

File: Data/Sys/GameSettings/R25.ini [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# R25EWR, R25PWR - LEGO Harry Potter: Years 1-4
+
+[Core]
+# Values set here will override the main Dolphin settings.
+
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId =
```
Comment (phire) [inline Data/Sys/GameSettings/R25.ini]: Emulation state here.

File: Data/Sys/GameSettings/R92.ini [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# R92E01, R92P01, R92J01 - Pikmin 2 Wii
+[Core]
+# Values set here will override the main Dolphin settings.
+[EmuState]
+# The Emulation State. 1 is worst, 5 is best, 0 is not set.
+EmulationStateId =
```
Comment (phire) [inline Data/Sys/GameSettings/R92.ini]: And here too.

**Comment by SMarioMan** [comment]
@phire Thanks for the tips. Fixed.

**Comment by phire** [comment]
It's going to need a rebase before it will build.

**Comment by SMarioMan** [comment]
@lioncash @phire I am attempting the rebase now. I'm new to this stuff so it took me awhile to figure out. Right now, the command line seems to be either slow or hanging, and I'm not sure why.

**Comment by SMarioMan** [comment]
@lioncash @phire I have rebased in a new pull request, since I couldn't resolve the file naming issue. 
https://github.com/dolphin-emu/dolphin/pull/2880

**Pull request closed without merging**